repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/cilanding.ftl

```diff
@@ -1,15 +1,72 @@
 <#ftl output_format="HTML">
 <#include "../include/imports.ftl">
+<#include "./macro/resource-image.ftl">
 <@hst.setBundle basename="publicationsystem.headers"/>
+<#assign formatFileSize="uk.nhs.digital.ps.directives.FileSizeFormatterDirective"?new() />
+
 <section class="document-content">
     <#if document??>
         <h1 data-uipath="ps.document.title">${document.title}</h1>
         <div class="layout layout--large">
-            <div class="layout__item layout-2-3">
-                <@hst.html hippohtml=document.content />
+            <div class="layout__item layout-2-3" data-uipath="ps.cilanding.sections">
+                <#if document.subSections?has_content>
+                    <p>
+                        <#list document.subSections as section>
+                            <#if section.title?has_content>
+                                <h2 data-uipath="ps.cilanding.section-title">
+                                    ${section.title}
+                                </h2>
+                            </#if>
+
+                            <#if section.content?has_content>
+                                <p data-uipath="ps.cilanding.content">
+                                    <@hst.html hippohtml=section.content />
+                                </p>
+                            </#if>
+
+                            <#if section.attachments?has_content>
+                                <section class="push-double--bottom">
+                                    <h2><@fmt.message key="headers.ci-landing-resources"/></h2>
+                                    <div class="panel panel--grey" data-uipath="ps.cilanding.attachments">
+                                        <#list section.attachments as attachment>
+                                            <div class="attachment media push-half--bottom">
+                                                <@resourceImage file=attachment.resource.filename />
+                                                <p class="media__body zeta">
+                                                    <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>">${attachment.text}</a> <span class="micro">&nbsp; [<@formatFileSize bytesCount=attachment.resource.length/>]</span>
+                                                </p>
+                                            </div>
+                                        </#list>
+                                    </div>
+                                </section>
+                            </#if>
+
+                            <#if section.assets?has_content>
+                                <p>
```

Comment:
no need for "<p>" here. 

Is that what we want? No grey box?

Reply:
Yes, links will be separate from the grey box which will only contain attachments.

Reply:
But "assets" and "attachments" are the same thing for end users, right?

Not sure if we should even support "attachments" in this content type.

Reply:
Assets can be (pre-uploaded) attachments, but also links to other CMS documents which the user selects on document completion.  I think as long as its ok for attachments uploaded as assets to be public before the document is actually published (which in this SHMI landing page case, it should be fine). I'll remove the Attachment compound type from this content type, and stick to assets instead.